README.md

```diff
@@ -38,9 +38,13 @@ Sounds great, how can I try this out?
 The following steps show how to build the examples:
 
 ```
+# first, install vcstools from PyPI or apt:
+# sudo apt install python3-vcstools
```

Comment:
The package name is `python3-vcstool`. `python3-vcstools` also exists, but it does a different thing, I know it's a bit confusing :-)

Reply:
Of course - sorry for the mix-up!

Reply:
No worries, thanks for addressing the feedback.